### DISCUSSION TITLE
[Idea] Make counter bubble optional

### DISCUSSION THREAD

**Discussion by adamlui** [badges/shields#9237] [Ideas] [answered]
Since the YouTube counter constantly shows 'invalid' due to [rate limiting](https://github.com/badges/shields/issues/8605#issuecomment-1455163387), a parameter to make shield counter bubbles optional would be quite useful (the YouTube one could be patched first since it's so popular)

**Comment by chris48s** [comment, marked as answer]
If you just want a static badge that doesn't show any number, you could use something like

https://img.shields.io/badge/Youtube-@AdamLuiTube-blue?style=social&logo=youtube&link=https://www.youtube.com/@AdamLuiTube

That said, I'm also going to try https://github.com/badges/shields/pull/9238 as a fix for the underlying issue.

**Reply by adamlui** [reply]
That is great!
